kombu/transport/pykafka.py

```diff
@@ -0,0 +1,336 @@
+"""
+kombu.transport.pykafka
```

Comment:
Bad bad Github! After updating with the rename change, the past comments vanished. But are still being count.

Alright.... rename done.